### PR TITLE
[PORT] v8

### PR DESCRIPTION
**Pull request by bealdav**
ping @hparfr @legalsylvain

### CODE REVIEW

File: proxy_action/__openerp__.py [hunk 1]
```diff
@@ -1,23 +1,6 @@
...
-#    You should have received a copy of the GNU Affero General Public License
-#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
-#
-##############################################################################
+# coding: utf-8
+# Copyright 2014 Sébastien BEAU <sebastien.beau@akretion.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
 
 {'name': 'Proxy Action',
  'version': '0.0.1',
```
Comment (legalsylvain) [inline proxy_action/__openerp__.py]: key : 8.0.1.0.0

File: proxy_action/helper.py [hunk 2]
```diff
@@ -1,42 +1,29 @@
...
-
-from openerp.osv import fields, orm
-from openerp import netsvc
+from openerp import api, models
 import base64
 
 
-class ProxyActionHelper(orm.AbstractModel):
-    _name="proxy.action.helper"
+class ProxyActionHelper(models.Model):
```
Comment (legalsylvain) [inline proxy_action/helper.py]: why not an abstract, finally ?

File: proxy_action/helper.py [hunk 3]
```diff
@@ -1,42 +1,29 @@
...
-        kwargs = {'options':{}}
+            host='https://localhost'):
+        """ Method to send send print action tu cups
+            In Odoo to populate pdf data use:
+            self.env['report'].get_pdf(object_ids, report_name)"""
+        if to_encode64:
+            data = base64.b64encode(data)
+        kwargs = {'options': {}}
         if copies > 1:
             kwargs['options']['copie'] = copies
```
Comment (legalsylvain) [inline proxy_action/helper.py]: Could you rename 'copie' in 'copies' ?

File: proxy_action/helper.py [hunk 4]
```diff
@@ -1,42 +1,29 @@
...
+            self, data,
+            printer_name=None,
             raw=False,
+            to_encode64=False,
             copies=1,
-            host='https://localhost',
-            context=None):
-        kwargs = {'options':{}}
+            host='https://localhost'):
+        """ Method to send send print action tu cups
```
Comment (hparfr) [inline proxy_action/helper.py]: send send tu

**Comment by legalsylvain** [comment]
Could you : 
- add a readme file ? 
- move description into -> the readme file.
- write the function you have deleted into the readme file, as a sample ?

:+1: Otherwise. Thanks !

File: proxy_action/helper.py (hunk 3 above)
Comment (bealdav) [inline proxy_action/helper.py]: pas compatible avec cups, j'imagine

Comment (hparfr) [inline proxy_action/helper.py]: C'est une erreur de portage
https://github.com/akretion/odooapps/blob/7.0/proxy_action/helper.py#L41

**Pull request merged**